source/functions/SimulinkModelFunctions/MoorDyn_matlab_FCN.m

```diff
@@ -0,0 +1,7 @@
+function FLines  = MoorDyn_matlab_FCN(X,XD,Time,CoupTime)
+
```

Comment:
This one is also has the outputs initialized prior to calling the function in the Simulink block. 

``` matlab

function FLines  = MoorDyn_matlab_FCN(X,XD,Time,CoupTime)

FLines = zeros(1,6);
FLines  = MoorDyn_matlab_FCN(X,XD,Time,CoupTime);

end

```